plugins/discourse-ai/app/controllers/discourse_ai/summarization/summary_controller.rb

```diff
@@ -41,6 +41,38 @@ def show
           end
         end
       end
+
+      def regen_gist
```

Comment:
Don't we want this to be async? Otherwise selecting a large number of topics will always be a timeout right ?

Maybe we schedule a `FastTrackTopicGist` for each ?